vector/fetch_vectors.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Script to fetch the hashes in `VECTOR_MANIFEST`, called as a part of `make deps`